test/unit/features/component/component-keep-alive.spec.js

```diff
@@ -862,5 +862,89 @@ describe('Component keep-alive', () => {
         )
       }).then(done)
     })
+
+    it('async components with transition-mode out-in', done => {
```

Comment:
I'm not familiar with writing tests for async components. Feel free to point me out If I'm not in the right way testing it.😅